ui/js/directives/treeherder/persona.js

```diff
@@ -64,7 +64,10 @@ treeherder.directive('personaButtons', [
                                 ThUserModel.get().then(function(user){
                                     angular.extend($rootScope.user, user);
                                 }, null);
-                            },function(){
+                            },function(response){
+                                var message = "Login failed: " + response.status + " " + response.statusText;
+                                thNotify.send(message, "danger", true);
```

Comment:
We could probably just put single quotes around `'danger'` since it's the internal notification style and not the customer message itself. Thanks for looking at this @sjakthol !